libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/DropInNavigationView.kt

```diff
@@ -80,6 +81,16 @@ class DropInNavigationView @JvmOverloads constructor(
         navigationContext.routeLineOptions = options
     }
 
+    /**
+     * Provide custom route arrow options to override the default options. This must be called
+     * before your activity or fragment's onStart() in order to take effect.
+     *
+     * @param options the [RouteArrowOptions] to use.
+     */
+    fun customize(options: RouteArrowOptions) {
+        navigationContext.routeArrowOptions = options
```

Comment:
(out of scope note)

Something that would be good to think about, is if we can make a mechanism that allows up you update these options at runtime. For example, could a developer change the route arrow at runtime?

Reply:
A good chunk of this question will be answered once we have NSS prototype ready